howso/react_series.amlg

```diff
@@ -317,6 +317,8 @@
 			context_values (null)
 			;{type "list" values "string"}
 			action_features (list)
+			;{type "list"}
```

Comment:
this is the old "batch_react_series" so needs to be a 2d list. I'd just copy the one for context_values, unless you know something I don't